views/editExistingSightingForm.ejs

```diff
@@ -0,0 +1,173 @@
+                <% } %>
+              <% } %>
+              <small id="vividDescription" class="form-text text-muted"
+                >Please enter your sighting as vividly as possible!</small
+              >
+            </div>
+            <div class="form-group">
+              <label for="date_time">Date & Time:</label>
+              <input
+                type="text"
+                class="form-control"
+                id="date_time"
+                placeholder="Date in DD/MM/YYYY HH:MM"
+                name="date_time"
+                value="<%=date_time%>"
+              />
+            </div>
+            <div class="form-group required">
+              <label class= "control-label" for="city">City:</label>
+              <input
+                type="text"
+                class="form-control"
+                id="city"
+                placeholder="E.g New York"
+                name="city"
+                value="<%=city%>"
+              />
+            </div>
+            <div class="form-group">
+              <label for="state">State:</label>
+              <input
+                type="text"
+                class="form-control"
+                id="state"
+                placeholder="E.g Nevada"
+                name="state"
+                value="<%=state%>"
+              />
+            </div>
+            <div class="form-group required">
+              <label class="control-label" for="shape">Shape:</label>
+              <input
+                type="text"
+                class="form-control
+                <%if(locals.feedback){%>
+                <% if(feedback.shape !=="ok"){ %>
+                 is-invalid 
+                <% } else { %> 
+                  is-valid 
+                <% } %>    
+                <% } %>"
+                id="shape"
+                placeholder="E.g Spherical"
+                name="shape"
+                value="<%=shape%>"
+              />
+              <%if(locals.feedback){%>
+                      <%if(feedback.shape){ %>
+                        <div class="invalid-feedback"><%= feedback.shape%></div>
+                      <% } %>
+                  <% } %>
+            </div>
+            <div class="form-group">
+              <label for="duration">Duration:</label>
+              <input
+                type="text"
+                class="form-control"
+                id="duration"
+                placeholder="E.g 3 minutes"
+                name="duration"
+                value="<%=duration%>"
+              />
+            </div>
+            <div class="form-group">
+              <label for="summary">Summary:</label>
+              <input
+                type="text"
+                class="form-control"
+                id="summary"
+                placeholder="Short overview of what you saw."
+                name="summary"
+                value="<%=summary%>"
+              />
+            </div>
```

Comment:
As form group logic gets more complex, e.g. with validations, consider creating an EJS sub-template for form groups.